### ISSUE TITLE
Store skills in genereal/[cat name]/[skill name] and include images

### ISSUE THREAD

**Issue by mariobehling** [fossasia/susi_server#1352]
The stored skill info is inconsistently stored. Images are not stored with the skill. Please store the skill text and the image in a subfolder of the chosen category to keep follow the approach of "seperating concerns".

Compare: https://github.com/fossasia/susi_skill_data/tree/master/models/general/Knowledge/en

**Comment by Orbiter** [comment]
I suggest that the folder for the skill get named like the current skill file name - without the '.txt' extension - and that the files inside the folder then have standard names, like "skill.txt" and "image.jpg"

**Comment by fragm3** [comment]
We need to store skills in a folder consisting of skill and image it is using. Check https://github.com/fossasia/susi_skill_data/tree/master/models/general/Communication/en, if the skill has a conflicting image name, the problem can arise due to the current structure. It would be better to store each skill in a separate folder consisting of text and image as pointed by @mariobehling

**Comment by mariobehling** [comment]
Good point. Though I am not certain if folder names support all kinds of names in all kinds of languages. So, let's follow the approach outlined by Orbiter. But also not that images can also be in other formats like png.

**Comment by mariobehling** [comment]
@fragm3 What is your idea for a solution if people create a skill with the same name?

**Comment by fragm3** [comment]
@mariobehling for that we require a database to store unique id for distinguishing between different skills
